### PR TITLE
 Update footer links to align with new Firefox products

### LINKED ISSUE #9089: Update footer links to align with new Firefox products
I was looking at the AMO footer and noticed that it links to Focus Browser, which is no longer promoted on mozilla.org. In fact, the link just goes to the mobile page.

I think we should update the AMO footer, replacing the Firefox section with the Browsers and Products sections in mozilla.org (making sure we still have UTM parameters on all of those links). Since that means 2 headers and more links, it's probably worth splitting into two columns (one for Browsers, one for Products and social media links).

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #9089

### CODE REVIEW

**Comment by championshuttler** [comment]
@jvillalobos Can you please provide UTM params for the links? Thanks

**Comment by jvillalobos** [comment]
External links from the footer should have these UTM params: `utm_source=addons.mozilla.org&utm_medium=referral&utm_content=footer-link`

File: src/amo/components/Footer/index.js [hunk 1]
```diff
@@ -143,6 +136,81 @@ export class FooterBase extends React.Component {
                 </a>
               </li>
             </ul>
+          </section>
+
+          <section className="Footer-product-links">
+            <h4 className="Footer-links-header">Product</h4>
```
Comment (diox) [inline src/amo/components/Footer/index.js]: This should plural - `Products` and be translated.

Comment (championshuttler) [reply src/amo/components/Footer/index.js]: @diox Done!

File: src/amo/components/Footer/index.js [hunk 2]
```diff
@@ -143,6 +136,81 @@ export class FooterBase extends React.Component {
...
+                  href={`https://www.mozilla.org/firefox/lockwise/${makeQueryStringWithUTM(
+                    {
+                      utm_source: 'addons.mozilla.org',
+                      utm_medium: 'referral',
+                      utm_content: 'footer-link',
+                      utm_campaign: null,
+                    },
+                  )}`}
+                >
+                  {i18n.gettext('Lockwise')}
```
Comment (diox) [inline src/amo/components/Footer/index.js]: Sorry about the back-and-forth, but I didn't catch this before:  `Products` and `Browsers` should be translated, as it is right now, but `Lockwise`, `Monitor`, `Send` and `Pocket` are product names and should *not* be translated.

**Comment by diox** [comment]
> External links from the footer should have these UTM params: `utm_source=addons.mozilla.org&utm_medium=referral&utm_content=footer-link`

@jvillalobos this doesn't match [what mozilla.org is using](https://github.com/mozilla/bedrock/blob/b479c3a9dbeb1490159c981686ebe53aeb837055/bedrock/base/templates/includes/protocol/footer/firefox.html#L33-L42) - they are using `utm_campaign=footer` and `utm_content=$productname`. Is that OK ?

**Comment by jvillalobos** [comment]
> @jvillalobos this doesn't match what mozilla.org is using - they are using utm_campaign=footer and utm_content=$productname. Is that OK ?

It's okay. Setting those params isn't an exact science and at the end of the day it just matters that we're sending the right data to track visits back to the AMO link.

**Pull request merged**